### ISSUE TITLE
how does vascular ectasia relate to aneurysm?

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4970]
@pnrobinson and @drseb- I talked to @matentzn today about aneurysms and vascular ectasia. We have two patterns for each of these:

https://github.com/obophenotype/upheno/blob/master/src/patterns/dosdp-workshop/aneurysm.yaml

https://github.com/obophenotype/upheno/blob/master/src/patterns/dosdp-workshop/vascularEctasia.yaml

Should these terms be synonyms (use the same patterns) or should they be different?

Here are some examples of aneurysm terms:

**HP_0004942 'Aortic aneurysm'**
'has part' some 
    ('increased amount'
     and ('inheres in' some 
        (aneurysm
         and ('part of' some aorta))))

**HP_0004944 'Dilatation of the cerebral artery'**
'has part' some 
    ('increased amount'
     and ('inheres in' some 
        (aneurysm
         and ('part of' some 'cerebral artery')))
     and ('has modifier' some abnormal))

But there are also some dilatation terms that use a different pattern:
**HP_0100860 'Dilatation of Inferior mesenteric artery'**
'has part' some 
    (ballooning
     and ('inheres in' some 'inferior mesenteric artery')
     and ('has modifier' some abnormal))

And according to ticket #1451, maybe we should be using a different pattern altogether (the one suggested in the ticket? I pasted it below)
```
'has part' some 
    ('increased amount'
     and fusiform
     and ('inheres in' some 
        (aneurysm
         and ('part of' some 'ascending aorta')))
     and ('has modifier' some abnormal))
```

There aren't actually any terms in HPo that use the vascular ectasia pattern.

**Comment by pnrobinson** [comment]
These are diffficult terms in the medical literature. To a first degree of approcimation, aneurysm is more severe than dilatation, and aneurysm also has subclasses (fusiform etc). Aneurym and dilation are used for relatively big blood vessels.

Some communities seem to prefer to use aneurysm and other dilatation and thus in the literature you may see both with the same (roughly) meaning.

 Ectasia can be used for small vessels including capillaries. We will need vascular ectasia I think and it is distinct from aneurysm.

We agreed with the cardiologists to use aneurysm as the primary label for the aorta

**Comment by pnrobinson** [comment]
@nicolevasilevsky  @matentzn Is there an action item for this?

**Comment by matentzn** [comment]
@pnrobinson we will come back to you; if there is no response before the 20th of January, you can close it. @nicolevasilevsky and I will determine the action items in our next meeting!

**Comment by nicolevasilevsky** [comment]
@pnrobinson I talked to Nico about this today, and we have some open questions.

It sounds like aneurysm and dilatation are used synonymously in the literature?

Question: is every aneurysm a dilatation?

if yes, we can create PATO terms for dilatation and a PATO term for aneurysm, and then created a pattern for these.

If not, we should just have a dilatation pattern.

Thanks!

**Comment by pnrobinson** [comment]
@nicolevasilevsky  The clinical terminology is not always consistent. For many people, Aneurysm sounds more severe than dilatation, although there is no agreement about where to draw the line. We have made the decision to use Aneurysm as the preferred label and dilatation as an exact synonym (for aortic aneurysm). For some other arteries, it seems that dilatation is more common.
 Aneurysm also has subclasses (fusiform, saccular) and these distinctions are not (or not often) used together with the word dilatation. 



I would recommend just having one pattern, which could be "dilated" or "aneurysmal". There should be subclasses of fusiform and saccular, see https://hpo.jax.org/app/browse/term/HP:0004959

**Comment by matentzn** [comment]
It seems then that the distinction really is more at the terminological level, and logically they are close enough (an EQ is less like a scalpel, and more like an ax). @nicolevasilevsky let's create then the abnormallyDilatedAnatomicalEntity and abnormallyDilatedAnatomicalEntityInLocation patterns and explicitly say how aneurysm is related in their definition.

**Comment by pnrobinson** [comment]
@nicolevasilevsky is this issue closable?

**Comment by matentzn** [comment]
I linked this to the respective  uPheno ticket so you can close it if you want!

**Comment by pnrobinson** [comment]
Thanks, Nico!
Here is a link to the uPheno ticket : https://github.com/obophenotype/upheno/issues/332
